x-pack/plugins/siem/public/components/netflow/index.test.tsx

```diff
@@ -120,7 +121,7 @@ const getNetflowInstance = () => (
 
 describe('Netflow', () => {
   test('renders correctly against snapshot', () => {
-    const wrapper = shallowWithIntl(<TestProviders>{getNetflowInstance()}</TestProviders>);
+    const wrapper = shallow(<TestProviders>{getNetflowInstance()}</TestProviders>);
```

Comment:
Can I understand it like we can apply `shallow` to all the snapshot test? But I am still not sure for rest of the cases, if we need to wrap the component with `TestProviders`, and if we need to mount / shallow with intl or not ...

Reply:
You kind of have to look at the output and see if it is going to help out or get in the way and if we exhaust all possibilities then just remove it and rely on the other unit tests to be helpful.
So I try in this order to see if I can get where I am going:
• shallow
• shallowInt
• shallow or shallowInt(<TestProvider>...)
• maybe mount ... or just remove it if it exceeds some 300+ lines and doesn't look helpful

In this example it looks like I probably should use `shallowWithIntl` with the component